tests/doc_format/embl/sequence/test_0008.xml

```diff
@@ -7,7 +7,7 @@
 
     <find-object-by-name index="kkk" doc="doc" name="AB000264" type="OT_SEQUENCE"/>
 
-    <check-sequence-alphabet-by-id obj="kkk" value="AMINO_DEFAULT_ALPHABET"/>
+    <check-sequence-alphabet-by-id obj="kkk" value="AMINO_EXTENDED_ALPHABET"/>
```

Comment:
Do not change the test. Make your algorithm case insensitive.

Reply:
I'ts not about case sensivity, i just edit wrong test.